cetmix_tower_server/models/cx_tower_server_template.py

```diff
@@ -200,6 +201,36 @@ def _create_new_server(self, name, **kwargs):
         Returns:
             cx.tower.server: newly created server record
         """
+        configuration_variables = kwargs.get("configuration_variables", {})
+        normalized_variables = {}
+        for variable, details in configuration_variables.items():
```

Comment:
What are you trying to do here and why?
Please add comments to your code otherwise it's barely understandable.